setup.py

```diff
@@ -66,7 +83,7 @@ def read(fname):
     name='studioml',
     version=VERSION,
     description='TensorFlow model and data management tool',
-    packages=find_packages(),
+    packages=find_packages(exclude=['tensorflow']),
```

Comment:
what does this change do? I thought `find_pacakges` searches for packages within the studio directory; but tensorflow is not one of those?

Reply:
It is good for situations where I include a proprietary body of code into my studio project directories.  One example being a proprietary runner, in another language, that has localized tensorflow test cases.  It is a NOP in the default studioml case.